options/options.html

```diff
@@ -144,275 +142,347 @@ <h4>
                 </div>
-                <div class="col-4">
+                <div class="col-3">
                     <label><input type="number" id="googletranslate-width" value="700">&nbsp;Width</label>
                 </div>
-                <div class="col-4">
+                <div class="col-3">
                     <label><input type="number" id="googletranslate-height" value="700">&nbsp;Height</label>
                 </div>
+                <div class="col-3">
+                    <label><input type="number" id="googletranslate-priority" value="22" min="1" step="1">Position</label>
+                </div>
             </div>
             <div class="row">
-                <div class="col-4">
+                <div class="col-3">
                     <label>Shaarli</label>
                 </div>
-                <div class="col-4">
+                <div class="col-3">
                     <label><input type="number" id="shaarli-width" value="700">&nbsp;Width</label>
                 </div>
-                <div class="col-4">
+                <div class="col-3">
                     <label><input type="number" id="shaarli-height" value="340">&nbsp;Height</label>
                 </div>
+                <div class="col-3">
+                    <label><input type="number" id="shaarli-priority" value="23" min="1" step="1">Position</label>
+                </div>
             </div>
             <div class="row">
-                <div class="col-4">
+                <div class="col-3">
                     <label>Mastodon</label>
                 </div>
-                <div class="col-4">
+                <div class="col-3">
                     <label><input type="number" id="mastodon-width" value="700">&nbsp;Width</label>
                 </div>
-                <div class="col-4">
+                <div class="col-3">
                     <label><input type="number" id="mastodon-height" value="340">&nbsp;Height</label>
                 </div>
+                <div class="col-3">
+                    <label><input type="number" id="mastodon-priority" value="24" min="1" step="1">Position</label>
+                </div>
             </div>
             <div class="row">
                 <div class="col-12">
                     <h1 class="center">Custom Shares</h1>
                     <p>Follow the placeholder and change only instance_url.<br>
                         <label>Shaarli: https://instance_url?post&title<br> <input type="url" id="shaarli" style="width:400px" placeholder="https://instance_url?post&title"></label><br>
-						<label>Mastodon: https://instance_url/share?text<br> <input type="url" id="mastodon" style="width:400px" placeholder="https://instance_url?post&title"></label>
+                        <label>Mastodon: https://instance_url/share?text<br> <input type="url" id="mastodon" style="width:400px" placeholder="https://instance_url?post&title"></label>
```

Comment:
That was a `\t` before.
Perhaps adding a [`.editorconfig`](https://editorconfig.org/) would help?

Reply:
let's do it :-D